src/control-plane/backend/lambda/api/service/reporting.ts

```diff
@@ -1361,11 +1366,6 @@ export class ReportingService {
         }
       }
 
-      //warm up quicksight
-      await quickSight.listDashboards({
-        AwsAccountId: awsAccountId,
-      });
-
```

Comment:
why remove this？

Reply:
During the warmup phase, only to establish **QuickSight** client connection, and this step [await getClickstreamUserArn](https://github.com/awslabs/clickstream-analytics-on-aws/blob/81e930c4d6791e84ffd075ac0f10541c1c8d6105/src/control-plane/backend/lambda/api/service/reporting.ts#L1323) already established the connection without the need for `listDashboards`.